### PR TITLE
Sheet Migration for Pricing PAhe

### PR DESCRIPTION
**Pull request by SAHU-01**
**Description**

This PR migrates to use a new data sheet for pricing page.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 75bb6c79930faee1d7e2e3be5cd3a72ff1349f28 at: https://675205c01f8b8903faf17d8e--layer5.netlify.app

File: src/sections/Pricing/generateOptions.js [hunk 1]
```diff
@@ -45,14 +44,12 @@ function generateOptions(data) {
...
   };
 
-  console.log("Tiers configuration:", tiers);
-
   const options = Object.entries(tiers).map(([tierName, tierInfo]) => {
-    console.log(`Processing tier: ${tierName}`, tierInfo);
 
     const summary = data
       .filter((item) => {
         const matches = item.subscription_tier === tierName && (item.pricing_page === "true" || item.pricing_page === "X" ) ;
```
Comment (leecalcote) [inline src/sections/Pricing/generateOptions.js:51]:
```suggestion
        const matches = item.subscription_tier === tierName && (item.pricing_page === "true" || item.pricing_page === "x" ) ;
```

At this point in the flow, I believe that we've shifted to lowercase (from the prior transformation in the workflow). Given that prudence we can skip another .toLowercase(), but do need to use a lowercase "x" in comparison.

**Comment by l5io** [comment]
🚀 Preview for commit 4239ec5ee30b5a27741964c2f64d6a9b9f230390 at: https://675209b03f2710223fa0226b--layer5.netlify.app

File: .github/workflows/feature-list.yml [hunk 2]
```diff
@@ -40,74 +40,48 @@ jobs:
...
-                    (pricingPage && ['x', 'X', 'true'].includes(pricingPage)) ||
-                    (documented.startsWith('https://docs.meshery.io/') ||
-                      documented.startsWith('https://docs.layer5.io/'));
-                  if (!includeRow) return null;
-
-                  return {
-                    theme: row['Theme (also: Keychain Name)'],
+                 const pricingPage = row['Pricing Page?']?.toLowerCase() || '';
+                 const hasXTier = ['Free Tier', 'TeamDesigner Tier', 'TeamOperator Tier', 'Enterprise Tier'].some(tier => row[tier]?.trim().toLowerCase() === 'x');
+                 const includRow = hasXTier || (pricingPage && ['x', 'X'].includes(pricingPage));
```
Comment (leecalcote) [inline .github/workflows/feature-list.yml:70]:
```suggestion
                 const includeRow = hasXTier || (pricingPage && ['x', 'X'].includes(pricingPage));
```

Missing an "e" here.

**Comment by l5io** [comment]
🚀 Preview for commit 9cf23cb2ccfe7dece06c63b84a57024a6fb750c9 at: https://67521ab97f2b993b846cf1f5--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 68850a3a3be7e2a6b91c91850bb2a090c87f5e06 at: https://675248c76570a5734e3a284a--layer5.netlify.app

File: spreadsheet.csv
Comment (leecalcote) [inline spreadsheet.csv:1]: This file can be deleted; doesn’t need checked in, Lee. Tsk, tsk.

File: src/sections/General/Navigation/navigation.style.js [hunk 3]
```diff
@@ -618,6 +618,14 @@ const NavigationWrap = styled.header`
     }
     .mobile-nav-item {
       padding: 1px;
+      UL:after {
```
Comment (leecalcote) [inline src/sections/General/Navigation/navigation.style.js:621]:
```suggestion
      ul:after {
```

**Comment by l5io** [comment]
🚀 Preview for commit b08179350aa53678d66c4d5c387deee63cffefe3 at: https://67527ee41f8b89963bf17d7a--layer5.netlify.app

**Pull request merged**